.travis.yml

```diff
@@ -11,27 +11,33 @@ os:
   - linux
   - osx
 
+sudo: required # needed for trusty
+dist: trusty # needed for Raqm
+
 before_install:
   - if [ "$TRAVIS_OS_NAME" = "linux" ];
     then
+      sudo apt-add-repository -y ppa:as-bahanta/raqm;
```

Comment:
i'm aware of the impact.  it's unfortunate.  but i don't think we want to make gd's test system dependent upon a random ppa.  please don't take this personally as i would have the same response for anyone who wanted to add a ppa to the list.